cli/formatting.go

```diff
@@ -0,0 +1,192 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	eos "github.com/eoscanada/eos-go"
+	"github.com/ryanuber/columnize"
+)
+
+func FormatPermissions(account *eos.AccountResp, config *columnize.Config) string {
+	output := []string{
+		"permissions:",
+	}
+	for _, perm := range account.Permissions {
+		keysValues := []string{}
+		for _, key := range perm.RequiredAuth.Keys {
+			keysValues = append(keysValues, fmt.Sprintf("%d %s", key.Weight, key.PublicKey))

```

Comment:
Seems we will need to update that at some point as it's possible to have other things than `Keys`, like `Accounts`.

Check https://eosquery.com/account/eoscanadacom compared to `eosc get account`:

```
permissions:
     active         4:
     blacklistops   1:  1 EOS7idX86zQ6M3mrzkGQ9MGHf4btSECmcTj4i8Le59ga7CpSpZYy5
     claimer        1:  1 EOS7NFuBesBKK5XHHLtzFxm7S57Eq11gUtndrsvq3Mt3XZNMTHfqc
     day2day        1:
     eosforumdapp   1:  1 EOS7YNS1swh6QWANkzGgFrjiX8E3u8WK5CK9GMAb6EzKVNZMYhCH3
     owner          4:  1 EOS5nFqz4giFz9kRZj6bByp9bgJQw7vEdrBAvMe7QWhr1xvX5YsTY, 1 EOS6EuWTngWDhzCwKoNxmrYnY87Pr8wPGYZ54ombRhQgpojqjavZX, 2 EOS7jFKsnSaA1BFdHBdhyFHHqHMdz4gcVG74WHtofeEGBiGLj4HCs, 1 EOS8CiR3Dpa1tHAY1yH8Tyf7Ue2JYRHKWAYMeo5gBHuHM2m8gKtBJ, 2 EOS8esiBart2LgGd98CwWgupqFMGtHxBV2uueesY5mHJNPRxCXCQQ
```

Reply:
Also, would it be possible for those with multiple keys to print them one under the above:

```
owner          4:  1 EOS5nFqz4giFz9kRZj6bByp9bgJQw7vEdrBAvMe7QWhr1xvX5YsTY
                   1 EOS6EuWTngWDhzCwKoNxmrYnY87Pr8wPGYZ54ombRhQgpojqjavZX
                   2 EOS7jFKsnSaA1BFdHBdhyFHHqHMdz4gcVG74WHtofeEGBiGLj4HCs
                   1 EOS8CiR3Dpa1tHAY1yH8Tyf7Ue2JYRHKWAYMeo5gBHuHM2m8gKtBJ
                   2 EOS8esiBart2LgGd98CwWgupqFMGtHxBV2uueesY5mHJNPRxCXCQQ
```

Reply:
Makes sense, can do that as well :)

Reply:
Do you want some sort of character/symbol to represent waits/accounts/keys?  I mean, it's pretty obvious without them

Reply:
@tampajohn it seems you dropped the `formatNestedPermissions` feature.. have you sync'd up with the latest code? I tweaked the permissions printout quite a bit..

Reply:
@abourget seems that I did when I was collapsing some of the merge commits, which explains the verbose commit history :).  I've added it back in and made it output a single line per value as @maoueh requested.